### PR TITLE
Training automation: step 1 and 2

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Downloading the vott json and images. We switched the order of the two because the endpoint /taggedimages will give us a list of images in completed tag state, and then we get those images from blob storage. 

Refactored code to move common code from cli/ to utils/

### CODE REVIEW

File: cli/training.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import os
+import requests
+from azure.storage.blob import BlockBlobService, ContentSettings
+from operations import (
+    read_config,
+    CONFIG_PATH,
+    get_azure_storage_client
+)
+
+class TagData(object):
```
Comment (andrebriggs) [inline cli/training.py:10]: @samiyaakhtar This should probably not live in the CLI area. Eventually we should move to a shared location.

File: cli/training.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            tagdata = TagData(data['imageUrls'][index], item, tag['tags'], tag['x1'], tag['x2'], tag['y1'], tag['y2'], tag['height'], tag['width'])
+            array_tags.append(tagdata)
+        vott_json[item] = array_tags
+        index += 1
+    
+    print(vott_json)
+    return vott_json
+            
+
+if __name__ == "__main__":
```
Comment (andrebriggs) [inline cli/training.py:64]: @samiyaakhtar is this for testing?

File: cli/training.py (hunk 1 above)
Comment (samiyaakhtar) [reply cli/training.py:10]: Okay, I can make the refactor happen as part of this PR.

File: train/training.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        vott_json[item] = array_tags
+        index += 1
+    
+    print(vott_json)
+    return vott_json
+            
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-n', '--num-images', type=int)
```
Comment (andrebriggs) [inline train/training.py]: I didn't know you could specify type, cool!

File: utils/blob_utils.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from azure.storage.blob import BlockBlobService, ContentSettings
+
+class BlobStorage():
```
Comment (andrebriggs) [inline utils/blob_utils.py]: @samiyaakhtar I think this is a good start. Eventually this code and the blob storage code the functions use should be the same. Ideally we may want to fully encapsulate "storage". Meaning that blob store specific operations live here and clients  (like training.py) don't "know" about blob storage. All the client knows is a save something or I retrieve something. This makes functionally testing a client (like training.py) somewhat easier. No action required now but just thinking about a possible future

File: utils/tagdata.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class TagData(object):
```
Comment (andrebriggs) [inline utils/tagdata.py]: @mtarng do you have a class like this in your branch of the db_access code?

**Review by andrebriggs — COMMENTED** [review]
@samiyaakhtar looks like the build is failing

File: utils/tagdata.py (hunk 5 above)
Comment (andrebriggs) [reply utils/tagdata.py]: @mtarng didn't realize you had a PR. Looking

File: cli/training.py (hunk 2 above)
Comment (samiyaakhtar) [reply cli/training.py]: I moved this file into train/

File: utils/blob_utils.py (hunk 4 above)
Comment (samiyaakhtar) [reply utils/blob_utils.py]: That sounds like a good idea, more abstraction for the training script should eventually be put into place

**Review by andrebriggs — APPROVED** [review]

**Pull request merged**